docs/src/main/docs/health/01_health.adoc

```diff
@@ -181,12 +182,31 @@ TIP: Balance collecting a lot of information with the need to avoid overloading
 
 === Built-in health-checks
 
-A set of built-in health checks can be optionally enabled to report various
- health check statuses that are commonly used:
+You can use Helidon-provided health checks to report various
+common health check statuses:
 
-* deadlock detection
-* available disk space
-* available heap memory
+[[built-in-health-checks-table]]
+[cols="1,1,3,20"]
```

Comment:
(optional) Table is ok as-is but you could add a column for default values and/or change the col size settings. Table renders nicely and resizes appropriately in current format.

Reply:
Fixed.